backend/src/v5/services/eventsListener/components/modelEvents.js

```diff
@@ -88,7 +88,7 @@ const revisionAdded = async ({ teamspace, project, model, revId, modelType, cali
 			{ _id: 0, tag: 1, author: 1, timestamp: 1, desc: 1, rFile: 1, format: 1, statusCode: 1, revCode: 1 });
```

Comment:
![image](https://github.com/user-attachments/assets/1904bbeb-2da4-431e-a0c0-383607c7f800)
We're missing an overflow strategy here, @apeitup  can you clarify what we should do here?

Reply:
we went for ellipsis with tool tip here... can we confirm with @apeitup that this is what we want?
![image](https://github.com/user-attachments/assets/888d1b93-88c7-4e39-bb50-1dade2e999d4)


@AsiteZane showed me a mockup of multiline instead when I mentioned it
![image](https://github.com/user-attachments/assets/866f97c9-6829-4eb9-a974-efb5ca7e2e2f)